src/JuMP.jl

```diff
@@ -424,36 +424,79 @@ function solve_time(model::Model)
 end
 
 """
-    set_parameter(model::Model, name, value)
+    set_optimizer_attribute(model::Model, name::String, value)
 
-Sets solver-specific parameter identified by `name` to `value`.
+Sets solver-specific attribute identified by `name` to `value`.
+
+This is equivalent to `set_optimizer_attribute(model, MOI.RawParameter(name), value)`.
+"""
+function set_optimizer_attribute(model::Model, name::String, value)
+    return set_optimizer_attribute(model, MOI.RawParameter(name), value)
+end
+
+"""
+    set_optimizer_attribute(
+        model::Model, attr::MOI.AbstractOptimizerAttribute, value
+    )
+
+Set the solver-specific attribute `attr` in `model` to `value`.
```

Comment:
We could give an example that `set_silent` is equivalent to `set_optimizer_attribute` with `MOI.Silent()`.